assets/javascripts/discourse/templates/components/tdc-sign-up.hbs

```diff
@@ -1,56 +1,68 @@
-{{#each site.group_names as |group|}}
-{{#if (is-collective group.full_name)}}
-<div class="user-field collective-selectors">
-  <div class="controls">
-    <label for="{{ group.name }}">
-    {{input type="checkbox" name="debt-type" id=group.name change=(action "collectiveSelected")}}
-    {{group.full_name}}
-    </label>
-  </div>
-</div>
-{{/if}} {{/each}}
+</div> <!-- again with the hacks ... hopefully this will fix the styling. The component is wrapped automatically in a div element, but that breaks the vanilla discourse styling. One can customize the component to be wrapped in another kind of html element, but as far as I can tell you can't just tell it not to wrap it at all -->
```

Comment:
😂 I've seen/done this before in other templating languages like Handlebars to wrap frustrating stuff in something reasonably selectable with CSS. No shame if it's what it takes.

Reply:
Is it being wrapped in the div just applying some weird spacing or something? Haven't actually looked to see what this looks like yet.

Reply:
I found an officialish solution: https://github.com/emberjs/ember.js/pull/4790